azure-pipelines-dependency-graph-testing.yml

```diff
@@ -153,10 +169,10 @@ stages:
               pip install -e dgtest
 
               # Run dgtest
-              dgtest run great_expectations --tests tests --branch 'origin/develop' --ignore 'tests/cli' --ignore 'tests/integration/usage_statistics' \
+              dgtest run great_expectations --ignore 'tests/cli' --ignore 'tests/integration/usage_statistics' \
```

Comment:
The `dgtest.ini` file in the root of the repo populates these defaults for us.

Since they are the same every time, leveraging a config file makes sense. Additionally, this will mean that developers that run `dgtest` locally will be able to pick up these values without any additional configuration.

```
# This basic command
dgtest run great_expectations --no-spark --no-postgresql

# gets expanded to this due to dgtest.ini
dgtest run great_expectations --tests tests --branch 'origin/develop' --no-spark --no-postgresql
```